src/test/java/org/takes/rq/RqFormTest.java

```diff
@@ -86,4 +86,49 @@ public void sameInstance() throws IOException {
             Matchers.is(Boolean.TRUE)
         );
     }
+
+    /**
+     * RqForm.Fake can create fake forms with parameters list.
+     * @throws IOException If something goes wrong.
+     */
+    @Test
+    public void createsFormRequestWithParams() throws IOException {
```

Comment:
@erimerturk I would use `throws Exception`, since it's more generic and it basically tells the reader that "whatever is thrown here - it's an error". not just `IOException`